ui/components/component-library/text-field/text-field.js

```diff
@@ -115,7 +114,7 @@ export const TextField = ({
       <InputComponent
         aria-invalid={error}
         as="input"
-        autoComplete={autoComplete ? 'on' : 'off'}
+        autoComplete={autoComplete}
```

Comment:
Since, InputComponent = Input already, do we need `as="input"` here?

Reply:
Yes! Resolved https://github.com/MetaMask/metamask-extension/pull/17664/commits/f3e19eea5648363b740c565208518b4ab8c8225b